src/tests/contract_test/core_test.cljs

```diff
@@ -0,0 +1,34 @@
+(ns tests.contract-test.core-test
```

Comment:
these tests are needed (or perhaps some of the requires are needed 🤔 ) to make the tests work.
I think the same for the integration tests.